docs/Running-behind-haproxy.md

```diff
@@ -7,8 +7,8 @@ custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/Running-beh
 
 # Netdata via HAProxy
 
-> HAProxy is a free, very fast and reliable solution offering high availability, load balancing, 
-> and proxying for TCP and HTTP-based applications. It is particularly suited for very high traffic web sites 
+> High Availability Proxy (HAProxy) is a free, very fast and reliable solution offering high availability, load balancing, 
```

Comment:
Again, given that HAProxy seems to be the official name, we don't want to mess with it.

```suggestion
> HAProxy (high-availability proxy) is a free, very fast and reliable solution offering high availability, load balancing, 
```